src/internal.c

```diff
@@ -12913,6 +12939,30 @@ static int ProcessPeerCertParse(WOLFSSL* ssl, ProcPeerCertArgs* args,
     }
 #endif
 
+#ifdef WOLFSSL_MAXQ10XX_TLS
+    /* MAXQ10xx accepts ASN_NO_SIGNER_E because the CA cert is registered with
```

Comment:
This seems like a case that should be coded into the cert manager logic, rather than carving out an exception here for it, but I'm not familiar with all the details.

Reply:
Not sure what that would look like.  I'm going to leave it as is for now.

Reply:
Huh, this logic isn't actually required as the maxq callback will always override the return value later.  Removing this logic.  Thanks for making me take a closer look.

Reply:
Removed in 2f06dede275d8c7d1924896d1ee358da840f0104